test/github-issues/5067/issue-5067.ts

```diff
@@ -0,0 +1,28 @@
+import "reflect-metadata";
+import {expect} from "chai";
+import {Connection} from "../../../src";
+import {closeTestingConnections, createTestingConnections} from "../../utils/test-utils";
+import {OracleDriver} from "../../../src/driver/oracle/OracleDriver";
+
+describe.only("github issues > #5067 ORA-00972: identifier is too long", () => {
```

Comment:
Aha, `describe.only` was set.  I'll undo that.

Reply:
Sorry, I forgot to remove after running the tests :/

Reply:
No worries! :) It happens.